python/cudf/cudf/__init__.py

```diff
@@ -101,10 +101,17 @@
 from cudf.core.tools.datetimes import date_range
 from cudf.utils.dtypes import _NA_REP
 from cudf.utils.utils import set_allocator
+from ptxcompiler.patch import patch_numba_codegen_if_needed
```

Comment:
Per Graham's question above should this be?

```suggestion
try:
    from ptxcompiler.patch import patch_numba_codegen_if_needed
except ImportError:
    pass
else:
    # Patch Numba to support CUDA enhanced compatibility.
    # See https://github.com/rapidsai/ptxcompiler for
    # details.
    patch_numba_codegen_if_needed()
    del patch_numba_codegen_if_needed
```